Dockerfile

```diff
@@ -61,9 +74,7 @@ ENV LANG=C.UTF-8 \
     SUPERSET_HOME="/app/superset_home" \
     SUPERSET_PORT=8088
 
-RUN mkdir -p ${PYTHONPATH} superset/static superset-frontend apache_superset.egg-info requirements \
-    && useradd --user-group -d ${SUPERSET_HOME} -m --no-log-init --shell /bin/bash superset \
```

Comment:
Here I'm thinking some of the funky commands may trigger the apt-get stuff to not use the cache at times. I see a lot of builds re-running `apt-get` when nothing here should have changed.